docs/docs/api/get.md

```diff
@@ -17,6 +17,8 @@ after the `view` or `producer` was triggered. It works the same way as
 2. A value is needed at a later time since producer was triggered, e.g while
    performing an asynchronous operation
 
```

Comment:
We might add a signature for Get of some sort, e.g.:

Get + Path -> (params?) => any // the value at the final Path